Sources/SpeziAccount/AccountService/Configuration/UserIdType.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 
 
 /// Determines the type and kind of the ``UserIdKey``.
-public enum UserIdType: Sendable, Equatable {
+public enum UserIdType {
```

Comment:
There seems to be an indentation problem here between line 17 & 18.

Reply:
This seems to be a display issue within GitHub. If you count the spaces in the browser manually it matches even.